rngb/Makefile

```diff
@@ -0,0 +1,10 @@
+#
+# Makefile for Phoenix-RTOS rngb
```

Comment:
maybe name the dir `random` or `misc` (from misc devices), as peripherals giving random numbers will have different names